index.js

```diff
@@ -1,12 +1,35 @@
 
+var defaults = require('lodash.defaults');
+
 var props = [
   'name',
   'message',
   'stack',
   'type',
 ]
 
-module.exports = function () {
+var DEFAULTS = {
+  format: function(err) {
+    var obj = {}
+
+    // set all enumerable properties of error onto the object
+    Object.keys(err).forEach(function (key) {
```

Comment:
Of course. But that bit was part of the original source and I wanted to make as little additions as possible.